analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -54,83 +56,175 @@ public static void VerifyNoExceptionThrown(string path, IEnumerable<DiagnosticAn
+                                          IEnumerable<MetadataReference> additionalReferences) =>
+            VerifyAnalyzer(paths, new[] { diagnosticAnalyzer }, null, CompilationErrorBehavior.Default, OutputKind.DynamicallyLinkedLibrary, additionalReferences);
+
+        public static void VerifyAnalyzer(string path,
+                                          DiagnosticAnalyzer[] diagnosticAnalyzers,
+                                          IEnumerable<ParseOptions> options = null,
+                                          CompilationErrorBehavior checkMode = CompilationErrorBehavior.Default,
+                                          OutputKind outputKind = OutputKind.DynamicallyLinkedLibrary,
+                                          IEnumerable<MetadataReference> additionalReferences = null) =>
+            VerifyAnalyzer(new[] { path }, diagnosticAnalyzers, options, checkMode, outputKind, additionalReferences);
 
         public static void VerifyAnalyzer(string path,
                                           DiagnosticAnalyzer diagnosticAnalyzer,
-                                          IEnumerable<MetadataReference> additionalReferences) =>
-            VerifyAnalyzer(new[] { path }, new[] { diagnosticAnalyzer }, null, CompilationErrorBehavior.Default, OutputKind.DynamicallyLinkedLibrary, additionalReferences);
+                                          IEnumerable<MetadataReference> additionalReferences,
+                                          string sonarProjectConfigPath) =>
+            VerifyAnalyzer(new[] { path },
+                           new[] { diagnosticAnalyzer },
+                           null, CompilationErrorBehavior.Default,
+                           OutputKind.DynamicallyLinkedLibrary,
+                           additionalReferences,
+                           sonarProjectConfigPath);
 
         public static void VerifyAnalyzer(string path,
                                           DiagnosticAnalyzer diagnosticAnalyzer,
                                           IEnumerable<ParseOptions> options,
                                           IEnumerable<MetadataReference> additionalReferences,
                                           string sonarProjectConfigPath = null) =>
-            VerifyAnalyzer(new[] { path }, new[] { diagnosticAnalyzer }, options, CompilationErrorBehavior.Default, OutputKind.DynamicallyLinkedLibrary, additionalReferences, sonarProjectConfigPath);
+            VerifyAnalyzer(new[] { path },
+                                     new[] { diagnosticAnalyzer },
+                                     options,
+                                     CompilationErrorBehavior.Default,
+                                     OutputKind.DynamicallyLinkedLibrary,
+                                     additionalReferences,
+                                     sonarProjectConfigPath);
```

Comment:
Messy indent. It's very expensive to maintain manually, I'd prefer invocations 4 on spaces.